scipy/integrate/odepack.py

```diff
@@ -257,3 +270,31 @@ def odeint(func, y0, t, args=(), Dfun=None, col_deriv=0, full_output=0,
         return output[0]
     else:
         return output
+
+
+def _make_empty_info(n):
+    """
+    Create an `info` dictionary for the return value of odeint in the
+    case where y0 has length 0 and full_output is True.
+    """
+    nm1 = n - 1
+    info = {
+        'hu': np.zeros(nm1, dtype=np.float64),
+        'tcur': np.zeros(nm1, dtype=np.float64),
+        'tolsf': np.zeros(nm1, dtype=np.float64),
+        'tsw': np.zeros(nm1, dtype=np.float64),
+        'nst': np.zeros(nm1, dtype=np.int32),
+        'nfe': np.zeros(nm1, dtype=np.int32),
+        'nje': np.zeros(nm1, dtype=np.int32),
+        'nqu': np.zeros(nm1, dtype=np.int32),
+        'imxer': -1,
+        'lenrw': 0,
+        'leniw': 0,
+        'mused': np.zeros(nm1, dtype=np.int32),
```

Comment:
Is a `mused` value of zero documented? (I see only 1 is adams, 2 is bdf.)

Reply:
I wanted to return an array with the expected shape and dtype for the given length of `t`, but I didn't think it was meaningful to use the values 1 or 2, since neither method was actually used.

All these arrays of zeros are just placeholders put there for consistency.  An alternative approach that is probably justifiable is to put the value `None` instead of an array.  The arrays provide information about the behavior of the solver, and since a solver never ran in this edge case, maybe we shouldn't provide these arrays at all.